llvm/test/CodeGen/AArch64/machine-outliner-overlap.mir

```diff
@@ -54,6 +55,27 @@ body:             |
     ; CHECK-NEXT: BL @OUTLINED_FUNCTION_0, implicit-def $lr, implicit $sp, implicit-def $lr, implicit-def $x8, implicit-def $x9, implicit $sp, implicit $x0, implicit $x9
     ; CHECK-NEXT: RET undef $x9
 
+    ; CHECK-LEAF-LABEL: name: overlap
```

Comment:
Can you add checks for the original case for the function `OUTLINED_FUNCTION_0` above so that we can compare different outlined functions?

Reply:
Added this.